spec/features/school_search_spec.rb

```diff
@@ -5,7 +5,7 @@
     claim = start_tslr_claim
     choose_qts_year
 
-    fill_in "School name", with: "Penistone"
```

Comment:
As the change you made in the previous commit broke the tests, I'd probably squash this commit into the previous one, with a note in the description about why you had to alter the tests

Reply:
Good call. Thank you.